src/hooks/useConnectCurrentVenue.ts

```diff
@@ -16,15 +16,15 @@ export const useConnectCurrentVenue = () => {
     venueId = getQueryParameters(window.location.search)?.venueId;
   }
 
-  useFirestoreConnect([
+  useSparkleFirestoreConnect([
     {
       collection: "venues",
       doc: venueId,
```

Comment:
I am pretty sure this could potentially be `undefined` based on reading it from `useParams` / `getQueryParameters `, which would cause a pathologic query that loads all venues.